Add AOCODARCF4V3

634296207 opened the pull request:
Add AOCODARCF4V3

RomanLut committed:
gps fix estimation squashed commit

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
included fixed USER3 BOX ID collision

RomanLut committed:
Merge branch 'submit-gps-fix-estimation' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
updated documentation

RomanLut committed:
retrigger checks

RomanLut committed:
retrigger checks

RomanLut committed:
retrigger checks

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
fixed rename getAirspeedEstimate()

RomanLut committed:
retrigger checks

RomanLut committed:
retrigger checks

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation
removed GPS Off box
implemented Disable GPS sensor fix logic condition

RomanLut committed:
Merge branch 'submit-gps-fix-estimation' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
adjusted docs

RomanLut committed:
fixed typo in comment

RomanLut committed:
added failsafe_gps_fix_estimation_delay

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Merge branch 'submit-gps-fix-estimation' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
enable possibility to simulate GPS sensor timeout, mag failure and pitot failure with HITL

RomanLut committed:
show pitot failure status on osd

RomanLut committed:
fix GPS timeout display on OSD

RomanLut committed:
estimate GPS Fix only if compass and barometer are healthy

RomanLut committed:
support GPS Fix estimation on sensor timeout

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
fixed compilation error

RomanLut committed:
Merge branch 'submit-gps-fix-estimation' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
Merge branch 'submit-simulator-sensor-failures' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
fixed indentation

RomanLut committed:
fixed indentation

RomanLut committed:
fixed indentation

RomanLut committed:
removed debug code

RomanLut committed:
attempt to fix failsafe_gps_fix_estimation_delay bug

RomanLut committed:
enable ground course estimation

RomanLut committed:
fixed landing detection with GPS fix estimation

RomanLut committed:
fixed: forced RTH is not activated on GPS Fix estimation event during mission with RX loss

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
fixed merge conflict

RomanLut committed:
fixed: groundspeed is not sent in mavlink telemetry

RomanLut committed:
use any wind estimator value with gps fix estimation

RomanLut committed:
improved course hold with GPS Fix estimation

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
fixed HITL HW sensors failure simulation

RomanLut committed:
fixed gpsSol access race conditions

RomanLut committed:
use wind vector for course correction on estimation on gps timeout

RomanLut committed:
fixed gps fix estimation on gps timeout

RomanLut committed:
return 99 satellites for logic condition on estimation

RomanLut committed:
removed redundand code

RomanLut committed:
allow course correction on estimation with gps sensor timeout

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
Merge branch 'submit-gps-fix-estimation' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
fixed compilation error

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Update GPS_fix_estimation.md

shota3527 committed:
back calculation anti-windup can only shrink integrator

shota3527 committed:
minor fix

RomanLut committed:
added USE_GPS_FIX_ESTIMATION
fixed indentation

RomanLut committed:
Merge branch 'submit-gps-fix-estimation' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
fixed unit test error

RomanLut committed:
ignore project file

wchnflr committed:
Create FLYWOOF405S_AIO

wchnflr committed:
Delete FLYWOOF405S_AIO

wchnflr committed:
Create CMakeLists

wchnflr committed:
Add files via upload

wchnflr committed:
Update target.h

wchnflr committed:
Update target.h

Changed MPU6000 and BMI270 gyro orientation from CW180 to CW270

wchnflr committed:
Update target.h

Added UART2
Bumped SERIAL_PORT_COUNT from 6 to 7
Changed TARGET_IO_PORTD mask from (BIT(2)) to 0xffff

wchnflr committed:
Update target.c

Fixed motor map to match Cinerace20 O3 so they don't need to be remapped in INAV Configurator.

wchnflr committed:
Update target.c

Remapped Motors 1-4 to match motor direction (Props In or Props Out) arrows in INAV Configurator.

wchnflr committed:
Update target.h

Changed TARGET_IO_PORTD mask from 0xffff to 0x0064 to strictly match PD5 and PD6 pins.

wchnflr committed:
Update target.c

Anothe rearrangement of motor assignments to match S1, S2, S3, S4 in Inav Configurator and actually fly properly.

wchnflr committed:
Update CMakeLists

Added SKIP_RELEASES tag

RomanLut committed:
allow  gps fix estimation without compass

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
Merge branch 'submit-gps-fix-estimation' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
fixed compilation error

wchnflr committed:
Changed TIM_USE_MC|FW_MOTOR to TIM_USE_OUTPUT_AUTO

RomanLut committed:
Merge branch 'master' of https://github.com/RomanLut/inav into submit-gps-fix-estimation

RomanLut committed:
fixed: OSD CMS menu does not work in SITL

shota3527 committed:
ahrs mag gps fusion

shota3527 committed:
minor fix

shota3527 committed:
development of posestimator fusion

shota3527 committed:
development of posestimator fusion

shota3527 committed:
development of posestimator fusion

shota3527 committed:
modify the weights

shota3527 committed:
Merge remote-tracking branch 'origin/sh_posz' into sh_ahrs

shota3527 committed:
update the imu heading estimation

shota3527 committed:
gps baro mag mix

shota3527 committed:
gps baro mag mix

shota3527 committed:
bug fixes on mag gps mix

shota3527 committed:
minor adjuestments

shota3527 committed:
adjuestable gps yaw weight

shota3527 committed:
update documents

shota3527 committed:
minor fix

shota3527 committed:
fix pos estimator acc weight

shota3527 committed:
update docs

shota3527 committed:
fixes display issue in configurator sensors tab

shota3527 committed:
Merge remote-tracking branch 'origin/master' into sh_ahrs

shota3527 committed:
Merge remote-tracking branch 'origin/master' into sh_ahrs

shota3527 committed:
minor fix on cog yaw estimation

add acc vibration to blackbox

adjustments in imu cog

minor fix on blackbox

shota3527 committed:
adjustments

RomanLut committed:
Merge branch 'master' of https://github.com/iNavFlight/inav into submit-gps-fix-estimation

shota3527 committed:
Merge remote-tracking branch 'origin/master' into sh_ahrs

MrD-RC committed:
Move minimum ground speed to a changeable parameter

MrD-RC committed:
Merge pull request #9427 from iNavFlight/master

Update from master

MrD-RC committed:
Update battery.c

Pump PG

DzikuVx committed:
Put getConfigMixerProfile as the last byte in message

DzikuVx committed:
Merge pull request #9431 from iNavFlight/fix-the-MSP2_INAV_STATUS-frame

Put getConfigMixerProfile as the last byte in message

breadoven committed:
Update navigation.c

MrD-RC committed:
Move parameter out of battery profile, in to normal settings.

breadoven committed:
Merge pull request #9435 from breadoven/abo_mc_nav_arm_safety_fix

Multirotor arming safety course hold/cruise mode fix

shota3527 committed:
sync updateActualHeading with other topic

RomanLut committed:
Merge branch 'master' of https://github.com/iNavFlight/inav into submit-gps-fix-estimation

RomanLut committed:
Update GPS_fix_estimation.md

RomanLut committed:
Update GPS_fix_estimation.md

shota3527 committed:
add more gps dynamic model

stronnag committed:
[GPS] fix GPS discovery for M6 and earlier

stronnag committed:
[GPS] update discovery fix for M7N as well

DzikuVx committed:
ATOMRCF405NAVI_DELUX first take

DzikuVx committed:
Merge pull request #9446 from iNavFlight/jh_fix_Neo6N_recognition

[GPS] fix  Neo6M recognition

DzikuVx committed:
IFLIGHT_BLITZ_ATF435 initial commit

mmosca committed:
Add DPS310 and SPL06 to FLYWOOF745 target

This was requested by flywoo support.

Also add other baros to match their internal target.

mmosca committed:
Merge pull request #9448 from iNavFlight/mmosca-flywoo-add-baro-flywoof745

Add DPS310 and SPL06 to FLYWOOF745 target (and other baros)

mmosca committed:
Remove PPM entry with conflicting timer.

Fixes #9442

mmosca committed:
cleanup

DzikuVx committed:
Various PWM output changes. Mainly to get rid of potential conflicts with new timer allocation

DzikuVx committed:
Merge remote-tracking branch 'origin/release_7.0.0' into dzikuvx-various-output-fixes

MrD-RC committed:
Show system message when linear descent begins

shota3527 committed:
Merge branch 'master' into sh_navpid_backcalc

MrD-RC committed:
Bug fix

Multiple RTH and early exit from RTH fixed. Message would only be displayed for first RTH.

DzikuVx committed:
change to ANYFCF7

DzikuVx committed:
Merge pull request #9452 from iNavFlight/dzikuvx-various-output-fixes

Various PWM output changes. Mainly to get rid of potential conflicts

shota3527 committed:
Add fade out effect to baro altitude correction

DzikuVx committed:
Change the SPI of Mamba H743

MrD-RC committed:
Merge pull request #9453 from iNavFlight/MrD_Add-message-when-transitioning-to-linear-descent

mmosca committed:
Change 3d deadband check to use 3d deadband min and max from ui, instead of mid_throttle_deadband

mmosca committed:
remove unused variable

mmosca committed:
Use correct method to read config

breadoven committed:
Update navigation_fixedwing.c

breadoven committed:
Merge pull request #9462 from breadoven/abo_circularloiter_fix

Fixed wing RC altitude adjustment fix

DzikuVx committed:
Merge pull request #9461 from iNavFlight/mmosca-3d-esc-arm-safety

Unable fo arm 3d esc rover

DzikuVx committed:
Merge remote-tracking branch 'origin/release_7.0.0' into dzikuvx-atomrc-f405-delux

DzikuVx committed:
Fix gyro orientation

DzikuVx committed:
Merge pull request #9464 from iNavFlight/dzikuvx-mambah743-updates

Change the SPI of Mamba H743

mmosca committed:
This board has an optional baro

Since the baro is not always populated, it needs more drivers.

DzikuVx committed:
Revert "Enable the virtual pitot by default"

This reverts commit 83844e996fbd229a16dc83e76e553fcf297ed5c8.

mmosca committed:
It looks like user actually had a v1 board

mmosca committed:
Change timer allocation default to MOTORS

This board has an unfortunate timer allocation, where the first 4 outputs use 3 timers.

This causes the default allocation to be M1, M2, M3, S1, M4

Fixes #9474

DzikuVx committed:
DAKEFPVF722 target

mmosca committed:
Add missing #include

DzikuVx committed:
DAKEFPVF405 target

mmosca committed:
Correct output order to match BF output order

https://github.com/betaflight/unified-targets/blob/master/configs/default/FLWO-FLYWOOF745.config

Reported by @sarcasticfpv on discord

sensei-hacker committed:
Flywoof745 of der commented motor numbers

Updates the // M1, // m2 etc to match the new order.

mmosca committed:
Fix motor labels on target.c and make sure M1-M4 land on S1-S4 by default

mmosca committed:
Merge pull request #9479 from sensei-hacker/patch-6

Flywoof745 of der commented motor numbers

mmosca committed:
Merge pull request #9475 from iNavFlight/mmosca-speedybeef7v3-bad-timer-allocation

Change speedybeF7V3 default timer allocation

DzikuVx committed:
Merge remote-tracking branch 'origin/release_7.0.0' into dzikuvx-atomrc-f405-delux

DzikuVx committed:
Update output assignment

error414 committed:
fix hdzero channel settings over displayport

mmosca committed:
Merge pull request #9478 from iNavFlight/mmosca-flywoof745-match-bf-outputs

Correct output order of FLYWOOF745 target to match BF and production board layout.

mmosca committed:
Merge pull request #9465 from iNavFlight/mmosca-tmotorf7v2-all-baro-drivers

TMOTORF7V2 has an optional baro, so add more drivers

DzikuVx committed:
Outputs update

DzikuVx committed:
Update current scale

DzikuVx committed:
Merge remote-tracking branch 'origin/release_7.0.0' into dzikuvx-atomrc-f405-delux

DzikuVx committed:
Merge pull request #9470 from iNavFlight/dzikuvx-disable-virtual-pitot

Revert "Enable the virtual pitot by default"

DzikuVx committed:
Merge pull request #9472 from iNavFlight/dzikuvx-atomrc-f405-delux

AtomRC F405 DELUX

mmosca committed:
Merge pull request #9251 from wchnflr/wchnflr-flywoof405s_aio

Add FLYWOO new unofficial target "FLYWOOF405S_AIO" by @wchnflr

sensei-hacker committed:
FLYWOOF405S_AIO: rename CMakeLists

mmosca committed:
Merge pull request #9494 from sensei-hacker/flywoo_aio_release_7.0.0

FLYWOOF405S_AIO: rename CMakeLists

DzikuVx committed:
Merge pull request #9496 from iNavFlight/dzikuvx-dakefpv-f722

DAKEFPVF722 target

DzikuVx committed:
Merge pull request #9495 from iNavFlight/dzikuvx-dakefpv-f405

DAKEFPVF405 target

DzikuVx committed:
Merge pull request #9486 from error414/hdzero-displayport-fix

fix hdzero channel settings over displayport

MrD-RC committed:
Add param to delay landing in event of a Failsafe

MrD-RC committed:
Update Settings.md

MrD-RC committed:
Bug fix

breadoven committed:
add BB wp number

634296207 committed:
Add AOCODARCF4V3

Add AOCODARCF4V3

634296207 committed:
Add AOCODARCF4V3

Add AOCODARCF4V3

aocodarc commented:
a) OK, we use AOCODARCF4V3 for the target, keep format consisitency with F4V2
b) Please do self test. Make sure self-test is passed, and add comment below

```
F405V3 Flash V2 self test passed.
```

lida2003 commented on src/main/target/AOCODARCF4V3/target.h:
```diff
@@ -0,0 +1,169 @@
+#define USE_PINIO
+#define USE_PINIOBOX
+#define PINIO1_PIN                  PC5  // VTX power switcher
+#define PINIO2_PIN                  PA14  //
```
Add comment for Bluetooth switcher please!

lida2003 replied:
Add comment for Bluetooth switcher, like VTX power switcher, please!

lida2003 commented on src/main/target/AOCODARCF4V3/target.h:
```diff
@@ -0,0 +1,169 @@
+// ******* FEATURES ********
+#define DEFAULT_RX_TYPE                 RX_TYPE_SERIAL
+#define SERIALRX_UART                   SERIAL_PORT_USART2
+#define SERIALRX_PROVIDER               SERIALRX_SBUS
```
I thought SERIALRX_SBUS / SERIALRX_CRSF is used for our AOC 7 inch PNP copters.

1. SERIALRX_SBUS for HD
2. SERIALRX_CRSF  for analog

I didn't do the math. If people like to use DJI RC more than ELRS then SERIALRX_SBUS , or SERIALRX_CRSF   should be set by default.

sensei-hacker committed:
Update Boards.md: Add link to full list

sensei-hacker committed:
Boards.md move punctuation

634296207 committed:
Update target.h

1. Add Bluetooth comment

2. Use the crsf protocol

3. Turn on the led light belt function

634296207 commented:
1.Add Bluetooth comment
2.Use the crsf protocol
2.Turn on the led light belt function

YI-BOYANG committed:
Add IIC2 and change the barometer to IIC2

MrD-RC committed:
Added landing delay message

MrD-RC committed:
Added precursor to future PR

Currently the standard system message function is blocked if any messages in the if section (osd.c:5108-5138) are used. I plan on unblocking the standard messages.

MrD-RC committed:
Additional Description for Control Derivative

MrD-RC committed:
added hint for BetaFlight users

DzikuVx committed:
Merge pull request #9510 from YI-BOYANG/master

GEPRCF405 add IIC2 and change the barometer to IIC2

vincentpoont2 committed:
Add ICM42688 to Kakutef7miniv3 target.h

Holybro will use ICM42688 to replace MPU6000 on this board.

jhemcu committed:
target: Add JHEMCU F722 targer board support

jhemcu committed:
target: Add JHEMCU F405 targer board support

mmosca committed:
Merge pull request #9522 from jhemcu/JHEMCUF722

target: Add JHEMCU F722 target board support.

DzikuVx committed:
flashhobby targets

mmosca committed:
Enable dshot

mmosca committed:
Merge pull request #9532 from iNavFlight/mmosca-betafpv435

Enable dshot for unofficial BETAFPVF435

mmosca committed:
Merge pull request #9437 from jhemcu/JHEMCUF405

target: Add JHEMCU F405 targer board support

DzikuVx committed:
SPEDIXF722 and SPEDIXF405 targets

DzikuVx committed:
Merge pull request #9536 from iNavFlight/dzikuvx-spedix-fcs

SPEDIXF722 and SPEDIXF405 targets

DzikuVx committed:
Merge pull request #9537 from iNavFlight/dzikuvx-flashhobby-fcs

flashhobby targets

DzikuVx committed:
Merge pull request #9538 from iNavFlight/dzikuvx-kakutef7mini3-gyro

Add ICM42688 to Kakutef7miniv3 target.h

mmosca committed:
Add VTX power info

mmosca committed:
Merge pull request #9539 from iNavFlight/mmosca-MAMBAH743_2022B.txt

Add VTX power info for MAMBAH743_2022B target

DzikuVx committed:
Kakute F4 V2.4

DzikuVx committed:
Merge pull request #9541 from iNavFlight/kakute-f405-v24

Kakute F4 V2.4

DzikuVx committed:
Merge pull request #9432 from iNavFlight/release_7.0.0

Release 7.0.0

DzikuVx committed:
Skip release for all F411 boards

DzikuVx committed:
INAV 8 version bump

MrD-RC committed:
Merge pull request #9521 from iNavFlight/MrD_Additional-Description-for-Control-Derivative

MrD-RC committed:
Merge pull request #9428 from iNavFlight/MrD_Make-min-ground-speed-a-parameter

MrD-RC committed:
Merge branch 'master' into MrD_Add-FS-Landing-delay

DzikuVx committed:
Add ICM42688P to KAKUTEH7

stronnag committed:
update f4 vcp in line with BF (#9544)

MrD-RC committed:
Merge pull request #9502 from iNavFlight/MrD_Add-FS-Landing-delay

Add a parameter to delay landing in the event of a Failsafe

DzikuVx committed:
Merge pull request #9546 from iNavFlight/dzikuvx-kakuteh7-icm42688

Add ICM42688P to KAKUTEH7

DzikuVx committed:
INAV 7.1

DzikuVx committed:
Merge pull request #9387 from shota3527/sh_ahrs

Yaw/Altitude estimation sensor fusion

DzikuVx committed:
Merge pull request #9224 from shota3527/sh_navpid_backcalc

Navpid back calculation anti-windup can only shrink integrator

DzikuVx committed:
Merge pull request #9507 from breadoven/abo_blackbox_add_active_wp

Add active waypoint number to blackbox

DzikuVx committed:
Merge pull request #9344 from RomanLut/submit-sitl-cms-menu

SITL: OSD CMS menu does not work

DzikuVx committed:
Merge pull request #8347 from RomanLut/submit-gps-fix-estimation

GPS Fix estimation (dead reconing, RTH without GPS fix) for fixed wing

DzikuVx committed:
Merge remote-tracking branch 'origin/master' into dzikuvx-7.1-mergeback-1

DzikuVx committed:
Merge pull request #9509 from sensei-hacker/boards_full_list

Boards.md: Add link to full list of boards

breadoven committed:
Update navigation.c

DzikuVx committed:
Merge pull request #9550 from iNavFlight/dzikuvx-7.1-mergeback-1

Dzikuvx 7.1 mergeback 1

DzikuVx committed:
Merge branch 'master' into dzikuvx-inav-8-version-bump

DzikuVx committed:
Merge pull request #9542 from iNavFlight/dzikuvx-skip-release-for-f411

Skip release for all F411 boards

DzikuVx committed:
Merge pull request #9543 from iNavFlight/dzikuvx-inav-8-version-bump

INAV 8 version bump

lida2003 commented:
Default settings, please keep consistent with https://github.com/Aocoda-RC/unified-targets/pull/2

breadoven committed:
Merge pull request #9551 from breadoven/abo_improve_wpmode_inhibit_control

WP mission mode RTH and Manual mode lockout improvements

634296207 committed:
Merge branch 'iNavFlight:master' into patch-1

634296207 committed:
Add config.c

1. Use the User1 and User2 modes

2. Serial port 5 Use Bluetooth

3. Serial port 2 Use a receiver

4. Serial port 3 Use power callback

5. Serial port 1 Use GPS

634296207 committed:
Update target.h

Bluetooth and 9v switches are enabled by default

634296207 commented:
> Default settings, please keep consistent with [Aocoda-RC/unified-targets#2](https://github.com/Aocoda-RC/unified-targets/pull/2）
I looked at someone else's code and couldn't find the Settings for dsoht600

634296207 commented:
I looked at someone else's code and couldn't find the Settings for dsoht600

lida2003 commented:
Can you cleanup the PR, as there are quite a lot of irrelevant commits?
